ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -664,432 +642,369 @@ describe('WorkspaceEdit', () => {
-
-    jest.useRealTimers();
-  });
-
-  it('shows confirmation on extended access delays', async () => {
-    workspaceEditMode = WorkspaceEditMode.Duplicate;
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-
-    workspacesApi.getWorkspace = (..._) => {
-      return Promise.resolve({
-        workspace,
-        accessLevel: WorkspaceAccessLevel.NO_ACCESS,
-      });
-    };
-
-    jest.useFakeTimers();
-    wrapper
-      .find('[data-test-id="review-request-btn-false"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-    wrapper
-      .find('[data-test-id="workspace-save-btn"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-    wrapper
-      .find('[data-test-id="workspace-confirm-save-btn"]')
-      .first()
-      .simulate('click');
-    let aclDelayBtn;
-    for (let i = 0; i < 10; i++) {
-      jest.advanceTimersByTime(20e3);
-      await waitOneTickAndUpdate(wrapper);
-      aclDelayBtn = wrapper
-        .find('[data-test-id="workspace-acl-delay-btn"]')
-        .first();
-      if (aclDelayBtn.exists()) {
-        break;
-      }
-    }
+    userEvent.click(
+      await screen.findByRole('button', {
+        name: /Duplicate Workspace/i,
+      })
+    );
 
-    if (!aclDelayBtn.exists()) {
-      fail(
-        'failed to find a rendered acl delay modal button after many timer increments'
-      );
-    }
-    expect(mockNavigate).not.toHaveBeenCalled();
+    const confirmSaveButton = await screen.findByRole('button', {
+      name: 'Confirm',
+    });
 
-    aclDelayBtn.simulate('click');
-    await waitOneTickAndUpdate(wrapper);
-    expect(mockNavigate).toHaveBeenCalled();
+    await user.click(confirmSaveButton);
+    await user.click(confirmSaveButton);
+    await user.click(confirmSaveButton);
 
-    jest.useRealTimers();
+    expect(workspacesApi.workspaceOperations.length).toEqual(numBefore + 1);
+    expect(mockNavigate).toHaveBeenCalledTimes(1);
   });
 
   it('should disallow whitespace-only intended study text', async () => {
-    const wrapper = component();
-    const intendedStudySection = wrapper.find(
-      '[data-test-id="intendedStudyText"]'
-    );
-    intendedStudySection
-      .find('textarea#intendedStudyText')
-      .simulate('change', { target: { value: '    ' } });
-
-    expect(getSaveButtonDisableMsg(wrapper, 'intendedStudy')).toBeDefined();
+    renderComponent();
+    const intendedStudySection = await screen.getByRole('textbox', {
+      name: /text area describing the intention of the study/i,
+    });
+    await user.type(intendedStudySection, '    ');
+    const saveButton = screen.getByRole('button', {
+      name: /create workspace/i,
+    });
+    await user.hover(saveButton);
+    // Testing to see if error appears
+    expect(
+      screen.queryByText(
+        /what are the specific scientific question\(s\) you intend to study \(question 2\.1\)/i
```

Comment:
is this the error message?  It looks like the question being asked - is there a good way to distinguish the question from the error?

Reply:
The question is:
What is the primary purpose of your project?

The error states:
What are the specific scientific question(s) you intend to study (Question 2.1)

I think the text difference is sufficient at distinguishing the two, but I open to other means if you have something in mind.

Reply:
ok I think this was my misunderstanding